src/components/o-datepicker-extended/_preview.html

```diff
@@ -13,21 +13,18 @@
 -->
 
 <div style="position: relative">
-  <axa-datepicker classes="o-datepicker js-datepicker" locale="de-ch"></axa-datepicker>
+  <axa-datepicker-extended classes="o-datepicker js-datepicker" locale="de-ch"></axa-datepicker-extended>
```

Comment:
hmh, why "extended"? extend from what, this seems semantically bad wording.

Did you care about this, because we have two components with same name atm?
Because I issued that already here #589 and would favour to have the child `m-datepicker` be renamed to `calender-sheet`

Reply:
I removed all datepicker changes. they were not meant to be in this PR.